src/js/components/Pagination/stories/Size.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Box, Grommet, Pagination, Text } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+export const Size = () => (
+  <Grommet theme={grommet}>
+    <Box align="start" pad="small" gap="medium">
+      <Box>
+        <Text>Default</Text>
+        <Pagination numberItems={237} />
+      </Box>
```

Comment:
I think we can remove this example, and be more explicit about 'Medium' as the default.

Reply:
Removed and incorporated recommendation.